quickjs.c

```diff
@@ -40493,38 +40493,31 @@ static const JSCFunctionListEntry js_math_obj[] = {
 
 /* OS dependent. d = argv[0] is in ms from 1970. Return the difference
    between UTC time and local time 'd' in minutes */
-static int getTimezoneOffset(int64_t time) {
+static int getTimezoneOffset() {
```

Comment:
```suggestion
static int getTimezoneOffset(void) {
```
Otherwise it's a function accepting any number of arguments (including none at all.)